extension/js/agent/utils/setHiddenProperty.js

```diff
@@ -1,10 +0,0 @@
-// @private
-var setHiddenProperty = function(object, property, value) {
-    if (!_.isObject(object)) return;
-    Object.defineProperty(object, hiddenPropertyPrefix+property, {
-        configurable: false,
-        enumerable: false,
-        value: value,
-        writable: true
-    });
-};
```

Comment:
I moved `setHiddenProperty` and `getHiddenProperty` into one file `hiddenProperty` since they both use that `hiddenPropertyPrefix` seemed to make sense to have them together